treeherder/model/migrations/0009_add_is_try_repo.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-03-09 18:59
```

Comment:
Ah ok. Typically we'd create it using:
`./manage.py makemigrations --name add_is_try_repo`